editor/editor_node.cpp

```diff
@@ -4120,10 +4131,119 @@ void EditorNode::update_ownership_table_for_addition_node_ancestors(Node *p_curr
 	}
 }
 
-void EditorNode::update_diff_data_for_node(
-		Node *p_edited_scene,
+void EditorNode::update_node_from_node_modification_entry(Node *p_node, ModificationNodeEntry &p_node_modification) {
+	if (p_node) {
+		// First, attempt to restore the script property since it may affect the get_property_list method.
+		StringName script_property_name = CoreStringNames::get_singleton()->_script;
```

Comment:
This is now:
```suggestion
		StringName script_property_name = CoreStringNames::get_singleton()->script;
```
Since `script_property_name` is only used once, there's little point in having this local variable though. Using `CoreStringNames::get_singleton()->script` directly should be as/more efficient, as it's already a pre-cached StringName.

Reply:
Duplicate of https://github.com/godotengine/godot/pull/92279#discussion_r1616166022.